pkg/v2/exec.go

```diff
@@ -22,9 +22,11 @@ type ExecTask struct {
 	// Args are the arguments to pass to the command. These are ignored if the
 	// Command contains arguments.
 	Args []string
-	// Shell run the command in a bash shell.
-	// Note that the system must have `/bin/bash` installed.
+	// Shell run the command in a (bash) shell.
 	Shell bool
+	// ShallPath is the path to the shell executable to use when Shell is true.
+	// If this is empty, the default `/bin/bash` is used.
+	ShallPath string
```

Comment:
I think that ShallPath should be ShellPath?

I'm going to try converting the module structure to match https://github.com/google/go-github

Then this PR could be sent again or rebased, for the shell path as discussed on #13, we want to try relying on Go to lookup the path itself